[CTOR-616] refresh Node Exporter Windows Metrics doc page (#3389)
centreon/centreon-documentation: pp/integrations/plugin-packs/procedures/applications-monitoring-node-exporter-windows.md

import Tabs from '@theme/Tabs';
import TabItem from '@theme/TabItem';

> 

Reviewer: ```suggestion

This pack helps you monitor a Windows server based on metrics returned by the Prometheus exporter.

```

## Pack assets

### Templates

The Monitoring Connector **Node Exporter Windows Metrics** brings a host template:

* **App-Monitoring-Node-Exporter-Windows-custom**

The connector brings the following service templates (sorted by the host template they are attached to):

<Tabs groupId="sync">
<TabItem value="App-Monitoring-Node-Exporter-Windows-custom" label="App-Monitoring-Node-Exporter-Windows-custom">

| Service Alias     | Service Template                                         | Service Description                 | Discovery  |
|:------------------|:---------------------------------------------------------|:------------------------------------|:----------:|
| Node-Cpu          | App-Monitoring-Node-Exporter-Windows-Cpu-custom          | Check node CPU usage                |            |
| Node-Memory       | App-Monitoring-Node-Exporter-Windows-Memory-custom       | Check node memory usage             |            |

> | Node-Service-Name | App-Monitoring-Node-Exporter-Windows-Service-Name-custom | Check services state                | X          |

Reviewer: ```suggestion
| Node-Service-Name | App-Monitoring-Node-Exporter-Windows-Service-Name-custom | Check the status of services | X          |
```

## Prerequisites

> This pack is based on community Prometheus exporter for Windows machines: https://github.com/prometheus-community/windows_exporter#installation.

Reviewer: ```suggestion
This pack is based on the community Prometheus exporter for Windows machines: https://github.com/prometheus-community/windows_exporter#installation.
```

## Installing the monitoring connector

### Pack

1. If the platform uses an *online* license, you can skip the package installation
instruction below as it is not required to have the connector displayed within the
**Configuration > Monitoring Connector Manager** menu.
If the platform uses an *offline* license, install the package on the **central server**
with the command corresponding to the operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-pack-applications-monitoring-node-exporter-windows
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-pack-applications-monitoring-node-exporter-windows
```

</TabItem>

> <TabItem value="Debian 11" label="Debian 11">

Reviewer: ```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

```

</TabItem>
</Tabs>

2. Whatever the license type (*online* or *offline*), install the **Node Exporter Windows Metrics** connector through
the **Configuration > Monitoring Connector Manager** menu.

### Plugin

Since Centreon 22.04, you can benefit from the 'Automatic plugin installation' feature.
When this feature is enabled, you can skip the installation part below.

You still have to manually install the plugin on the poller(s) when:
- Automatic plugin installation is turned off
- You want to run a discovery job from a poller that doesn't monitor any resource of this kind yet

> More information in the [Installing the plugin](/docs/monitoring/pluginpacks/#installing-the-plugin) section.

Use the commands below according to your operating system's package manager:

<Tabs groupId="sync">
<TabItem value="Alma / RHEL / Oracle Linux 8" label="Alma / RHEL / Oracle Linux 8">

```bash
dnf install centreon-plugin-Applications-Monitoring-Nodeexporter-Windows
```

</TabItem>
<TabItem value="Alma / RHEL / Oracle Linux 9" label="Alma / RHEL / Oracle Linux 9">

```bash
dnf install centreon-plugin-Applications-Monitoring-Nodeexporter-Windows
```

</TabItem>

> <TabItem value="Debian 11" label="Debian 11">

Reviewer: ```suggestion
<TabItem value="Debian 11 & 12" label="Debian 11 & 12">
```

</Tabs>

## Using the monitoring connector

### Using a host template provided by the connector

1. Log into Centreon and add a new host through **Configuration > Hosts**.
2. Fill in the **Name**, **Alias** & **IP Address/DNS** fields according to your resource's settings.
3. Apply the **App-Monitoring-Node-Exporter-Windows-custom** template to the host. A list of macros appears. Macros allow you to define how the connector will connect to the resource, and to customize the connector's behavior.
4. Fill in the macros you want. Some macros are mandatory.

| Macro             | Description                                                                                          | Default value     | Mandatory   |
|:------------------|:-----------------------------------------------------------------------------------------------------|:------------------|:-----------:|
| NODEEXPORTERPROTO | Specify https if needed (default: 'http')                                                            | http              |             |
| NODEEXPORTERURL   | URL to scrape metrics from (default: '/metrics')                                                     | /metrics          |             |

> | NODEEXPORTERPORT  | Port used (default: 80)                                                                              | 9182              |             |

Reviewer 2: ```suggestion
| NODEEXPORTERPORT  | Port used                                                                              | 9182              |             |
```
